.gitignore

```diff
@@ -18,4 +18,8 @@ RemoteSystemsTempFiles/
 Certs/
 .vscode/
 *~
-agrirouter_ids.json
\ No newline at end of file
+build/*
```

Comment:
You don't need to do this for Meson's sake: https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Reply:
My Meson version is older than 0.57.0. 
I use the old version, because I can not update, due to the development on an embedded system